skin/src/main/res/drawable/rss_ic_circle_empty_24dp.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+
+    <size
+        android:width="24dp"
```

Comment:
We may want to make this a dimension.  An alternate way to do this, that would scale, is to use a ShapeDrawable programattically